.eslintignore

```diff
@@ -0,0 +1,23 @@
+_cdn
+_site
+docs/.jekyll-metadata
+node_modules
+npm-debug.log*
+yarn-error.log*
+.DS_Store
+.history
+yarn.lock
+package-lock.json
+.github
+.storybook
+*.min
+lint.log
+dist
+*.bundle
+scripts
+gulpfile.js
+*.bundle.js
+*.bundle.map.js
+*.stories.js
```

Comment:
I think we should check stories.js. There could be syntax or indentation problems there. 
Lets figure out how to fix that export error